### PR TITLE
Add localization to date picker options

### PR DESCRIPTION
**Pull request by RedouaneM**
Hello, 

After many tries to set localization without success, I decided to add an item in the DatePickerOptions. This solution doesn't need any additional injection. 

Could you please, accept my PR?

Gratefully

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,13 +42,15 @@ export class AppModule {}
 
 ```ts
 import { DatepickerOptions } from 'ng2-datepicker';
+import * as frLocale from 'date-fns/locale/fr';
```
Comment (jkuri) [inline README.md]: is there a reason why we should have `fr` hardcoded?

Comment (RedouaneM) [reply README.md]: @jkuri 
No, it's just an example. It could be any other language code.

Comment (RedouaneM) [reply README.md]: @jkuri 
Could you please accept my PR ?

**Comment by jkuri** [comment]
merged and published. thanks!

**Pull request merged**